src/amo/containers/Search.js

```diff
@@ -0,0 +1,5 @@
+import SearchResult from 'amo/components/SearchResult';
+import SearchPageBase from 'core/containers/SearchPage';
+
+
+export default SearchPageBase(SearchResult);
```

Comment:
I think this is what I'm doing wrong, but forgive me I'm TOTALLY lost as to what the right approach is here. 😢